### ISSUE TITLE
Final chapter green lightning gloves stops working after save-load

### ISSUE THREAD

**Issue by SG5** [bfeber/HLA-NoVR#143]
In the final chapter in first lock room, once you get green lightning gloves, I killed all enemies and started destroing green locks.
Once I saved game only 2 locks left to be destroyed, and game crushed. After loading the save file, green lightning stoped working and I can't finish the game

**Comment by SG5** [comment]
I was able finish first room after I loade save file just after you get green gloves.
But I got same problem (green lightning gloves stops working after save-load) in next room, where you have to throw a lot of grenades. After green wave I got the gloves charged, but save-load breaks it

**Comment by riverscent** [comment]
I've experienced the same issue. For me it occurs after a death and when the last save loads, even an autosave, the gloves stop working. The game does not crash nor does Steam crash. The first time it happened, I thought it was part of the game and I had to finish off the remaining enemies using the grenades available on the dead ones. Then, I was just stuck in the first room. 

There was other odd behavior when this occurred. Two of the grunts (white suits) had no weapons even thought their hands were positioned as if they held one threatening to shoot. They do not attack. If you go near them, they physically strike and kick you. (I thought perhaps with all the geometry and gravity weirdness earlier in the chapter that perhaps this was a normal David Lynchian world and spent a good number of minutes trying to get out of the room while leaving them alone and then longer once I killed even them. LOL)
